appcli/git_repositories/git_repositories.py

```diff
@@ -42,7 +44,7 @@ def init(self):
 
         # Confirm that a repo doesn't already exist at this directory
         if self.repo_exists():
-            error_and_exit(
+            raise Exception(
                 f"Cannot initialise repo at [{self.repo_path}], already exists."
```

Comment:
discuss

Reply:
Discussed - essentially I want to capture the Exception error messages so that I can group them together and display more than one at a time.